owncloudApp/src/main/res/layout/account_item.xml

```diff
@@ -16,111 +16,120 @@
   You should have received a copy of the GNU General Public License
   along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/constraintLayout"
     android:layout_width="match_parent"
     android:layout_height="80dp"
-    xmlns:app="http://schemas.android.com/apk/res-auto">
+    android:background="?android:attr/selectableItemBackground"
+    >
 
     <ImageView
         android:id="@+id/icon"
         android:layout_width="40dp"
         android:layout_height="40dp"
-        android:layout_alignParentStart="true"
-        android:layout_centerInParent="true"
-        android:layout_marginStart="@dimen/standard_margin"
+        android:scaleType="fitXY"
         android:src="@drawable/ic_account_plus"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toStartOf="@+id/name"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
         />
 
     <ImageView
         android:id="@+id/ticker"
         android:layout_width="16dp"
         android:layout_height="16dp"
-        android:layout_alignEnd="@id/icon"
-        android:layout_alignTop="@id/icon"
-        android:layout_marginEnd="-8dp"
-        android:layout_marginTop="-8dp"
         android:src="@drawable/ic_current"
-        />
+        tools:layout_editor_absoluteX="48dp"
+        tools:layout_editor_absoluteY="13dp"
+        android:layout_marginTop="-8dp"
+        android:layout_marginEnd="-8dp"
```

Comment:
🤔 This is not convincing for me... Couldn't you get the same result with some paddings maybe? Negative margins are not a good practice 😵

Reply:
I've been reading about this and found that negative margins were added in the 2021 version because it was an easiest way to let an object position over other object, so if it's not big deal i'd prefer to let this stay like that